### PR TITLE
added api endpoint for delete contact

### PR DESCRIPTION
**Pull request by thdvbr**

### CODE REVIEW

File: src/routes/sosContactRoutes.js [hunk 1]
```diff
@@ -55,13 +55,36 @@ router
       (err) => {
         if (!err) {
           res.send('Successfully added contact');
+          console.log(req.body);
```
Comment (irina-baeva) [inline src/routes/sosContactRoutes.js]: console logs

File: src/routes/sosContactRoutes.js [hunk 2]
```diff
@@ -55,13 +55,36 @@ router
...
+        contacts: {
+          _id: req.query.id,
+        },
+      },
+    },
+    { safe: true },
+    (err) => {
+      if (!err) {
+        res.send('Successfully deleted Contact');
+        console.log(req.query);
```
Comment (irina-baeva) [inline src/routes/sosContactRoutes.js]: and here console log

File: src/routes/sosContactRoutes.js [hunk 3]
```diff
@@ -55,13 +55,36 @@ router
...
+          console.log(req.body);
         } else {
           res.send(err);
         }
       },
     );
   });
 
+router.delete(`/users/:username/delete`, (req, res) => {
+  Contact.updateOne(
```
Comment (irina-baeva) [inline src/routes/sosContactRoutes.js]: not clear from route what u going to delete, seems like user, but it is contact? so when i create delete user option, whould be overlapping 
there is good explanation in the first part of the article:  https://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api#restful

Comment (thdvbr) [reply src/routes/sosContactRoutes.js]: changed to '/users/:username/contacts' for now

**Pull request merged**